src/grappler/ngraph_optimizer.h

```diff
@@ -22,6 +22,7 @@
 #include "tensorflow/core/graph/graph.h"
 #include "tensorflow/core/graph/graph_constructor.h"
 #include "tensorflow/core/graph/graph_def_builder.h"
+#include "tensorflow/core/graph/node_builder.h"
```

Comment:
With this upgrade is `skip_these_nodes` needed?
```
Status MarkForClustering(Graph* graph,
                         const std::vector<string> skip_these_nodes) {
```

Reply:
Yes, keeping it around for future changes like we discussed.